### PR TITLE
Migration:adding manual installation steps for ubuntu to doubtfire io

### PR DESCRIPTION
**Pull request by jeevanloyd**
i have added two files. It includes manual installation for Ubuntu
ubuntu
ubuntu_web

.

### CODE REVIEW

File: source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+[4. Install Doubtfire API dependencies](#Install_Doubtfire_API_dependencies)
+
+[5. Create and populate Doubtfire](#Create_and_populate_Doubtfire)
+
+[6. Install LaTeX to generate PDFs](#Install_LaTeX_to_generate_PDFs)
+
+---
+<a id="Install_rbenv_and_ruby_build"></a>
+#####Install rbenv and ruby-build
```
Comment (thisaraw) [inline source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md:40]: Please add numbering like the below example.
**1. Install rbenv and ruby-build**

File: source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+Now install Ruby v2.3.1:
+
+```
+$ sudo apt-get install -y libreadline-dev
+$ rbenv install 2.3.1
+```
+---
+
+<a id="Install_postgres"></a>
+#####Install Postgres
```
Comment (thisaraw) [inline source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md:72]: Please add numbering.

File: source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+#####Install LaTeX to generate PDFs
+
+Follow the [Generating PDFs](https://github.com/doubtfire-lms/doubtfire-api/wiki/Generating-PDFs) guide to assist with installing LaTeX to generate PDFs. This step is optional unless you wish to generate PDF submissions.
+
+
+You can clone this wiki locally by
+
+```
+https://github.com/doubtfire-lms/doubtfire-api.wiki.git
+```
```
Comment (thisaraw) [inline source/articles/installation/ubuntu/2020-04-21-installation-guides-for-doubtfire-web.html.md:165]: Could you please let me know why did you add these lines?

File: source/articles/installation/ubuntu/index.html.md.erb [hunk 4]
```diff
@@ -12,7 +12,7 @@ There are multiple ways to deploy Doubtfire to an Ubuntu development environment
 
 - [Pre-packaged Virtual Machine](./virtual-machine/)
 - [Automated Installation Scripts](./installation-scripts/)
-- [Manual Installation](./manual-installation/)
+- [Manual Installation](./2020-04-21-installation-guides-for-doubtfire-web/)
```
Comment (thisaraw) [inline source/articles/installation/ubuntu/index.html.md.erb:15]: Why is it written installation-guides-for-doubtfire-web?
It is for all 3 applications, isn't it?

Comment (jeevanloyd) [reply source/articles/installation/ubuntu/index.html.md.erb:15]: Actually i had created guide for web . But as per requirement I had to put changes in a single file.

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
+---
+
+title: Ubuntu Manual Installation
+date: 2020-04-21 15:11 UTC
+tags: installation,ubuntu
+
+author: Jeevan loyd Fernandes
+summary: check to see the installation
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:8]: Please update the summary to reflect the contents of the article.

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+
+
+---
+you can install
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:12]: Make this more active maybe something like "Use these steps to install"

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+```
+<strong>Note:</strong> if you're using [Oh-My-Zsh](https://ohmyz.sh/), add to your `.zshrc`:
+
+```
+$ echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.zshrc
+$ echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+$ echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.zshrc
+```
+
+Now install Ruby v2.3.1:
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:72]: This can be updated now - check the gem file for the matching version.

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 8]
```diff
@@ -0,0 +1,268 @@
+---
+
+title: Ubuntu Manual Installation
+date: 2020-04-21 15:11 UTC
+tags: installation,ubuntu
+
+author: Jeevan loyd Fernandes
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:7]: Is this really all your work? Please note original authors

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 9]
```diff
@@ -0,0 +1,268 @@
...
+$ echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.zshrc
+$ echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+$ echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.zshrc
+```
+
+Now install Ruby v2.3.1:
+
+```
+$ sudo apt-get install -y libreadline-dev
+$ rbenv install 2.3.1
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:76]: Also here

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 10]
```diff
@@ -0,0 +1,268 @@
...
+
+<strong>Note:</strong>You may need to install `grunt-cli` globally in Linux using sudo.
+
+Lastly, to compile and run a watch server and web server, use `npm start`:
+
+```
+$ npm start
+```
+
+This will automatically run the angular 1 `grunt watch`, and the angular 7 `ng serve`.
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:223]: You can update this now to the current process.

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md [hunk 11]
```diff
@@ -0,0 +1,268 @@
...
+$ rbenv rehash
+```
+
+Then, install [bundler](https://bundler.io/) to sort out your ruby dependencies:
+
+```
+$ gem install bundler
+$ bundle install
+```
+
```
Comment (macite) [inline source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:264]: How do you run and test?

File: source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md (hunk 8 above)
Comment (jeevanloyd) [reply source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:7]: Hi Andrew, this file was created by me and I have worked on adding contents for ubuntu.
Earlier there was a file created by Jacob Taylor, but I didn't use it, instead, I created a new file.
If I am wrong, could I please know how can I add the author?

Comment (macite) [reply source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:7]: Its ok - you just need to be careful with things like this. This is just a text file... you can edit the text to change who you are saying is an author. Maybe look at the original source and see who was the author there.

Comment (jeevanloyd) [reply source/articles/installation/ubuntu/manual-installation-guides-for-doubtfire-for-ubuntu.html.md:7]: Yes sure. Thank you for getting back Sir. I will do changes as per the comment.
